qa/L0_backend_python/examples/test.sh

```diff
@@ -37,9 +37,9 @@ rm -fr *.log python_backend/
 # Install torch
 pip3 uninstall -y torch
 if [[ "$TEST_JETSON" == "0" ]] || [[ ${TEST_WINDOWS} == 0 ]]; then
-    pip3 install torch==2.0.0+cu117 -f https://download.pytorch.org/whl/torch_stable.html torchvision==0.15.0+cu117
+    pip3 install torch==2.2.0+cu121 -f https://download.pytorch.org/whl/torch_stable.html torchvision==0.16.0+cu121
```

Comment:
This output of `pip3 list` from `nvcr.io/nvidia/pytorch:24.01-py3` image  shows that `torchvision` version needs to be higher 
```
pytorch-quantization      2.1.2
torch                     2.2.0a0+81ea7a4
torch-tensorrt            2.2.0a0
torchdata                 0.7.0a0
torchtext                 0.17.0a0
torchvision               0.17.0a0
```